source/layouts/partials/_meta.html.erb

```diff
@@ -60,13 +67,10 @@
 <meta http-equiv="X-UA-Compatible" content="IE=edge">
 <meta name="viewport" content="width=device-width,initial-scale=1">
 
-<% if config[:env_name] != 'production' # don't index or follow non-production pages %>
-  <meta name="robots" content="NoIndex, NoFollow">
-<% elsif metas[:no_index] # explicitly don't index this page %>
-  <meta name="robots" content="NoIndex">
-<% else %>
-  <meta name="robots" content="Index, Follow">
+<% if config[:env_name] != 'production'  %>
+  <!-- [<%= config[:env_name] %> override] If merged into production, this would be: "<%= raw_robots_content %>" -->
```

Comment:
This is an HTML comment to help us debug…I see no other way as non-production must never be indexed.